desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.scss

```diff
@@ -106,7 +106,6 @@
   .hue-path-browser__input {
     width: 300px;
     flex: 0 0 auto;
-    // height: 32px;
 
     .ant-input-prefix {
       width: 24px;
```

Comment:
I assume this is for the icon, if so then using the $icon-width variable would make it much more readable and easier to maintain